docs/usage/validators.md

```diff
@@ -64,6 +64,51 @@ _(This script is complete, it should run "as is")_
 You'll often want to use this together with `pre`, since otherwise with `always=True`
 *pydantic* would try to validate the default `None` which would cause an error.
 
+## Reuse validators
+
+Occasionally, you will want to use the same validator on multiple fields/models (e.g. to
+normalize some input data). The "naive" approach would be to write a separate
+classmethod for each field. E.g.:
+
+```py
+{!.tmp_examples/validators_allow_reuse.py!}
+```
+_(This script is complete, it should run "as is")_
+
+Obviously, this entails a lot of repetition and boiler plate code. To circumvent this,
+the `allow_reuse` parameter has been added to `pydantic.validator` in version 1.2
+(`False` by default):
+
+```py
+{!.tmp_examples/validators_allow_reuse2.py!}
+```
+_(This script is complete, it should run "as is")_
+
+As it is obvious, repetition has been reduced and the models become again almost
+declarative.
+
+!!! note
+    In case you find the syntax strange - it is indeed unusual! - you should [read a bit
```

Comment:
I don't think we need this, also I don't think just linking to a google search is helpful. It feels a bit passive aggressive like lmgtfy